app/src/main/scala/org/alephium/explorer/api/UtilsEndpoints.scala

```diff
@@ -28,8 +28,23 @@ trait UtilsEndpoints extends BaseEndpoint with QueryParams {
       .tag("Utils")
       .in("utils")
 
+  private val logLevels    = List("TRACE", "DEBUG", "INFO", "WARN", "ERROR")
+  private val logLevelsStr = logLevels.mkString(", ")
+
   val sanityCheck: BaseEndpoint[Unit, Unit] =
     utilsEndpoint.put
       .in("sanity-check")
       .description("Perform a sanity check")
+
+  val changeGlobalLogLevel: BaseEndpoint[String, Unit] =
+    utilsEndpoint.put
+      .in("update-global-loglevel")
+      .in(plainBody[String].validate(Validator.enumeration(logLevels)))
+      .description(s"Update global log level, accepted: $logLevelsStr")
+
+  val changeLogConfig: BaseEndpoint[String, Unit] =
+    utilsEndpoint.put
+      .in("update-log-config")
+      .in(plainBody[String])
+      .description("Update logging file, only logback.xml is accepted")
```

Comment:
`update-global-loglevel` looks nice and validated 👍🏼. Just confirming that `update-log-config` is safe? (access to totally update a config file thats doing disk IO).

Reply:
it only updates the file in runtime, it doesn't change the  disk version. Maybe we should use another endpoint path?

Reply:
Oh I should've been more clearer. I didn't mean the disk version. Just meant to ask if it's safe to allow total web access to a config file that configures logging IO (something that does disk operations) which is only validated by the logback's parser?

I have no idea. When it comes to security, it favours coding defensively so in the past for similar situations I've had the config file sitting somewhere outside on the same server and load it on boot-up. But maybe that's just me being paranoid. 

Configuring logging via web API is very nice and convenient and if it's safe in terms of security then this looks great 👍🏼 🚀

Reply:
>config file that configures logging IO (something that does disk operations)

Oh I haven't thought of that. We are restricting the access to those endpoints in our load balancer, but still it might not be a good pratice. Any opinion @killerwhile ?

Maybe we could have a safer version where you can only pass a list of values:
```
[
  {
    "name": "akka.http.impl.engine.client",
    "level":"WARN"
    }
]
```